Gopkg.toml

```diff
@@ -33,3 +33,7 @@
 [[override]]
   name = "github.com/golang/protobuf"
   version = "1.1.0"
+
+[[constraint]]
```

Comment:
why this have no impact on Gopkg.lock and vendor packages?

Reply:
my mistake, resolved it